pages/glossary/clearly-labeled-location.md

```diff
@@ -7,11 +7,11 @@ objective: false
 
 Secondary information and alternative controls of functionality are often not displayed together with primary information or functionality. For example, an option to change a web page to dark mode may be placed on an options page instead of being available on every page and page state of a website. Another example is a maps application, where, instead of using GPS, an option is available in a dropdown menu to set the current location of the device. Such content should be placed in a clearly labeled location.
 
-The location of a target is said to be _clearly labeled_ when the target can be found by activating [instruments][] whose "description" either leads the user to find the target, or to another [page][web page] or page state from which this action can be repeated until the target is found.
+The location of a target is said to be _clearly labeled_ when the target can be found by activating "identifiable" [instruments][] which either lead the user to find the target, or to another [page][web page] or page state from which this action can be repeated until the target is found.
 
-Whether or not the content is "clearly labeled" depends on the starting point of the search. If page A has a link with a clear "description" to some piece of content, then the location of the content is clearly labeled. Page B, which can be in the same website, may not have such a link or may have a link with an ambiguous link text, and so the location of the content starting from page B is not clearly labeled.
+Whether or not the content is "clearly labeled" depends on the starting point of the search. If page A has a link which clearly "identifies" some piece of content, then the location of the content is clearly labeled. Page B, which can be in the same website, may not have such a link or may have a link with an ambiguous link text, and so the location of the content starting from page B is not clearly labeled.
 
-For the purpose of this definition, a _description_ is any text or other content with a [text alternative][], that allows any user to identify an element with a [semantic role][] that inherits from `widget`.
+For the purpose of this definition, an [instrument][] is _identifiable_ if any text or other content with a [text alternative][], allows any user to identify an element with a [semantic role][] that inherits from `widget`.
```

Comment:
This looks better.
Still on the description/identification issue:
> Page B, which can be in the same website, may not have such a link or may have a link **with an ambiguous link text**

What does "ambiguous link text" means here? I undertand it as "non-descriptive" which, we agreed, has nothing to do with the definition.

Reply:
I've tried to expand how "ambiguous link text" should be interpreted here. What do you think?